CHANGELOG.md

```diff
@@ -1,6 +1,104 @@
+
+
+### Build System Changes 
+
+- Export `Kokkos_CXX_COMPILER_VERSION` [\#6282](https://github.com/kokkos/kokkos/pull/6282)
+- Rename AMD GPU Architectures to align with AMD preferences: [\#6266](https://github.com/kokkos/kokkos/pull/6266)
+- Remove calling `tribits_exclude_autotools_files()` [\#6247](https://github.com/kokkos/kokkos/pull/6247)
+- Disable default oneDPL support in Trilinos [\#6342](https://github.com/kokkos/kokkos/pull/6342)
+
+### Incompatibilities (i.e. breaking changes)
+ - Ensure that `Kokkos::complex` only gets instantiated for cv-unqualified floating-point types  [\#6251](https://github.com/kokkos/kokkos/pull/6251)
+ - Removed (deprecated-3) support for volatile join operators in reductions [\#6385](https://github.com/kokkos/kokkos/pull/6385)
+ - Enforce `ViewCtorArgs` restrictions for `create_mirror_view` [\#6304](https://github.com/kokkos/kokkos/pull/6304)
+ - SIMD types for ARM NEON are not autodetected anymore but need `Kokkos_ARCH_ARM_NEON` or `Kokkos_ARCH_NATIVE=ON` [\#6394](https://github.com/kokkos/kokkos/pull/6394)
+ - Remove `#include <iostream>` from headers where possible [\#6482](https://github.com/kokkos/kokkos/pull/6482)
+
+### Deprecations
+- Deprecated `Kokkos::vector` [\#6252](https://github.com/kokkos/kokkos/pull/6252)
+- All host allocation mechanisms except for `STD_MALLOC` have been deprecated [\#6341](https://github.com/kokkos/kokkos/pull/6341)
+
+### Bug Fixes
+ - Fix AVX2 SIMD support for ZEN2 AMD CPU [\#6238](https://github.com/kokkos/kokkos/pull/6238)
+ - Missing memory fence in `RandomPool::free_state` functions [\#6290](https://github.com/kokkos/kokkos/pull/6290)
+ - Fix for corner case in `is_partitioned` [\#6257](https://github.com/kokkos/kokkos/pull/6257)
+ - Fix initialization of scratch lock variables in the `Cuda` backend [\#6433](https://github.com/kokkos/kokkos/pull/6433)
+ - Fixes for `Kokkos::Array` [\#6372](https://github.com/kokkos/kokkos/pull/6372)
+ - Fixes for LLVM-Clacc for OpenACC [\#6240](https://github.com/kokkos/kokkos/pull/6240) and [\#6250](https://github.com/kokkos/kokkos/pull/6250)
+ - Fixed symlink configure issue for Windows [\#6241](https://github.com/kokkos/kokkos/pull/6241)
+ - Fix cudaAPI wrapper errors for `CUDA_MALLOC_ASYNC` [\#6346](https://github.com/kokkos/kokkos/pull/6346)
+ - simd: added a missing generator ctor for avx2 float simd_mask [\#6397](https://github.com/kokkos/kokkos/pull/6397)
+ - Use execution space instance argument to get device properties in block size deduction [\#6419](https://github.com/kokkos/kokkos/pull/6419)
+ - OpenMPTarget init-join fix [\#6444](https://github.com/kokkos/kokkos/pull/6444)
+ - Fix atomic operations bug for Min and Max [\#6435](https://github.com/kokkos/kokkos/pull/6435)
+ - Fix implementation for `cyl_bessel_i0` [\#6484](https://github.com/kokkos/kokkos/pull/6484)
+ - Fix various NVCC warnings [\#6483](https://github.com/kokkos/kokkos/pull/6483)
```

Comment:
```suggestion
 - Fix various NVCC warnings in `BinSort` [\#6483](https://github.com/kokkos/kokkos/pull/6483)
```

Reply:
The pull request touches 5 files, so we have `BinSort`, `algorithms` tests, `Array`, `BitManipulation`, and `MathematicalFunctiobns` tests.

Reply:
These unit tests fixes are not worth mentioning in the changelog